third_party/gfxreconstruct/android/tools/replay/src/main/resources/lib/arm64-v8a/wrap.sh

```diff
@@ -15,11 +15,26 @@
 # limitations under the License.
 
 # GOOGLE: this script is used for preloading libwrap for PM4 capture
+# The rest is to allow for debugging. See https://developer.android.com/ndk/guides/wrap-script
+
+cmd=$1
+shift
+
 capture_pm4_enabled=$(getprop debug.dive.replay.capture_pm4)
+os_version=$(getprop ro.build.version.sdk)
+
+if [ "$os_version" -eq "27" ]; then
+  cmd="$cmd -Xrunjdwp:transport=dt_android_adb,suspend=n,server=y -Xcompiler-option --debuggable $@"
+elif [ "$os_version" -eq "28" ]; then
+  cmd="$cmd -XjdwpProvider:adbconnection -XjdwpOptions:suspend=n,server=y -Xcompiler-option --debuggable $@"
+else
+  # Unlike what the docs claim, need -Xcompiler-option --debuggable here as well
+  cmd="$cmd -XjdwpProvider:adbconnection -XjdwpOptions:suspend=n,server=y -Xcompiler-option --debuggable $@"
+fi
```

Comment:
Since these branches are the same, consider collapsing these into a single statement
```suggestion
else
  # Despite what the Android docs claim, need -Xcompiler-option --debuggable here as well
  cmd="$cmd -XjdwpProvider:adbconnection -XjdwpOptions:suspend=n,server=y -Xcompiler-option --debuggable $@"
fi
```